tools/cldr-apps/js/src/esm/cldrNotify.mjs

```diff
@@ -48,6 +52,9 @@ function open(message, description) {
  * @param {String} description the more detailed description
  */
 function warning(message, description) {
+  if (hasDataDog) {
+    datadogLogs.logger.warn(message, { description });
+  }
```

Comment:
is it going to be too noisy to ship all of these?

Reply:
I guess we'll find out -- hasDataDog will be false on production until/unless we enable it after testing on cldr-staging, right?

Reply:
correct